content/influxdb/v2.0/backup-restore/backup.md

```diff
@@ -19,9 +19,9 @@ InfluxDB copies all data and metadata to a set of files stored in a specified di
 on your local filesystem.
 
 {{% warn %}}
-#### InfluxDB 2.0 (release candidate)
+#### InfluxDB 2.0
 
-The `influx backup` command is not compatible with InfluxDB 2.0 (release candidate).
+The `influx backup` command is not compatible with InfluxDB 2.0.
```

Comment:
1.x.... not 2.0.

Reply:
Perhaps you should say, this command is not compatible with earlier versions of InfluxDB 1.x or InfluxDB 2.x releases prior to GA?